frontend/src/pages/Groups/GroupPage/gql.ts

```diff
@@ -64,6 +64,17 @@ export const POST_DISCUSSION_MESSAGE = gql`
   }
 `;
 
+export const LEAVE_GROUP = gql`
+  mutation leaveGroup($user: String!, $group: String!) {
+    leaveGroup(
+      userId: $user
+      groupId: $group
+    ) {
+      success
+    }
```

Comment:
Idk if its possible, but if we could actually return an http error instead of the success message, then we could catch the errors in the frontend in a `try/catch` block. Is that possible?